src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```
@@ -208,7 +208,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </Target>
 
   <Target Name="ResolvePackageAssets"
-          Condition=" '$(DesignTimeBuild)' != 'true' Or Exists('$(ProjectAssetsFile)')"
+          Condition=" ('$(DesignTimeBuild)' != 'true' Or Exists('$(ProjectAssetsFile)' ) ) And '$(SkipResolvePackageAssets)' != 'true'"
```

```suggestion
          Condition="('$(DesignTimeBuild)' != 'true' Or Exists('$(ProjectAssetsFile)')) And '$(SkipResolvePackageAssets)' != 'true'"
```

extra internal whitespace